KUnit error: arch/riscv/mm/dma-noncoherent.c:54: Error: unrecognized opcode cbo.clean (a0)', extension zicbom' required

**Environment:** Pioneer box SG2042

**Branch:** OLK-6.6

**Command Executed:** `./tools/testing/kunit/kunit.py run --arch=riscv`

**Error:** 
```
$ ./tools/testing/kunit/kunit.py run --arch=riscv
[11:28:56] Configuring KUnit Kernel ...
[11:28:56] Building KUnit Kernel ...
Populating config with:
$ make ARCH=riscv O=.kunit olddefconfig
Building with:
$ make ARCH=riscv O=.kunit --jobs=64
ERROR:root:../arch/riscv/mm/dma-noncoherent.c: Assembler messages:
../arch/riscv/mm/dma-noncoherent.c:54: Error: unrecognized opcode `cbo.clean (a0)', extension `zicbom' required
../arch/riscv/mm/dma-noncoherent.c:68: Error: unrecognized opcode `cbo.inval (a0)', extension `zicbom' required
../arch/riscv/mm/dma-noncoherent.c:155: Error: unrecognized opcode `cbo.flush (a0)', extension `zicbom' required
../arch/riscv/mm/dma-noncoherent.c:61: Error: attempt to move .org backwards
../arch/riscv/mm/dma-noncoherent.c:75: Error: attempt to move .org backwards
../arch/riscv/mm/dma-noncoherent.c:162: Error: attempt to move .org backwards
make[5]: *** [../scripts/Makefile.build:243: arch/riscv/mm/dma-noncoherent.o] Error 1
make[4]: *** [../scripts/Makefile.build:480: arch/riscv/mm] Error 2
make[3]: *** [../scripts/Makefile.build:480: arch/riscv] Error 2
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [/home/xxx/kernel/Makefile:1924: .] Error 2
make[1]: *** [/home/xxx/kernel/Makefile:234: __sub-make] Error 2
make: *** [Makefile:234: __sub-make] Error 2
```

--- a/arch/riscv/include/asm/errata_list.h
+++ b/arch/riscv/include/asm/errata_list.h
@@ -166,11 +166,11 @@ asm volatile(ALTERNATIVE_2(						\
 	"mv a0, %1\n\t"							\
 	"j 2f\n\t"							\
 	"3:\n\t"							\
-	"cbo." __stringify(_op) " (a0)\n\t"				\
+	CBO_##_op(a0)							\
 	"add a0, a0, %0\n\t"						\
 	"2:\n\t"							\
 	"bltu a0, %2, 3b\n\t"						\
-	"nop", 0, CPUFEATURE_ZICBOM, CONFIG_RISCV_ISA_ZICBOM,		\
+	"nop", 0, RISCV_ISA_EXT_ZICBOM, CONFIG_RISCV_ISA_ZICBOM,		\
 	"mv a0, %3\n\t"							\
 	"j 2f\n\t"							\
 	"3:\n\t"							\

